setup.py

```diff
@@ -54,7 +54,7 @@ def run(self):
         'Werkzeug >= 0.11',
     ),
     extras_require={
-        'jwt': ('PyJWT >= 1.4.0',),
+        'jwt': ('PyJWT >= 1.4.0', 'cryptography >= 2.0.3'),
```

Comment:
Maybe `>= 2.0.0`? All Python deps are peer deps, so it's nice to use round-ish numbers.